tx-pool/src/process.rs

```diff
@@ -729,6 +731,18 @@ impl TxPoolService {
                 .is_vm_version_1_and_syscalls_2_enabled(epoch)
         };
 
+        // log tx verification result for monitor node
+        if log_enabled_target!("ckb_tx_monitor", Trace) {
+            if let Ok(c) = ret {
+                trace_target!(
+                    "ckb_tx_monitor",
+                    r#"{{"tx_hash":"{}","cycles":{}}}"#,
+                    H256::from_slice(tx_hash.as_slice()).unwrap(),
```

Comment:
```suggestion
                    r#"{{"tx_hash":"{:#x}","cycles":{}}}"#,
                    tx_hash,
```

Reply:
remove lossy codes in chain and tx-pool, Thx.